arch/arm/include/irq.h

```diff
@@ -76,6 +77,17 @@
 
 #ifndef __ASSEMBLY__
 
+#ifndef up_switch_context
+#define up_switch_context(tcb, rtcb)                              \
```

Comment:
why not use inline function?

Reply:
If we use inline, we need to see the definition of struct tcb_s, but irq.h can't include sched/sched.h